app/Http/Requests/TimeSheetRequest.php

```diff
@@ -31,8 +31,7 @@ public function rules()
     public function messages()
     {
         return [
-            'date' => "date is required!"
+            'date' => array('required', 'Date'),
```

Comment:
This entire file is unnecessary, don't form validate a get request.

I'm fixing this in my branch so just remove this file